web/views.py

```diff
@@ -28,13 +57,17 @@ def get(self):
             else:
                 user = User.get(telegram_id=callback_state)
                 if user:
-                    user.spotify_id = user_creds.id
+                    user.spotify_id = user_creds.id or "FIXME"
```

Comment:
I would like to draw your attention to this separately @buurro FYI

I discovered that on my local database, which was generated from the ORM, the field was constrained as NOT NULL, which made it impossible to register.

You may want to not include this in the pull request, or you may also include it because this ID has no value anywhere for now and this placeholder does not affect anything.

Then, later this can be handled using the appropriate migrations

Reply:
Thanks for letting me know